frontend/src/core/comments/components/AddComment.js

```diff
@@ -0,0 +1,90 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from '@fluent/react';
+
+import './Comment.css';
+
+import { UserAvatar } from 'core/user'
+
+type Props = {|
+    user: string,
+    username: string,
+    imageURL: string,
+|};
+
+
+export default function AddComments(props: Props) {
+    const { user, username, imageURL } = props;
+
+    if (!user) {
+        return null;
+    }
+
+    const submitComment = (event: SyntheticMouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        let comment=document.querySelector('#comment-input').value
```

Comment:
You should use a [Ref](https://reactjs.org/docs/refs-and-the-dom.html#adding-a-ref-to-a-dom-element) instead. Check out how we do that in [Machinery](https://github.com/mozilla/pontoon/blob/master/frontend/src/modules/machinery/components/Machinery.js).